<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPublication</class>
 <widget class="QWidget" name="AddPublication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Publicação</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainWidget">
    <item>
     <widget class="QLabel" name="name_lbl">
      <property name="text">
       <string>Nome</string>
      </property>
      <property name="buddy">
       <cstring>name</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name"/>
    </item>
    <item>
     <widget class="QLabel" name="year_lbl">
      <property name="text">
       <string>Ano</string>
      </property>
      <property name="buddy">
       <cstring>year</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="year"/>
    </item>
    <item>
     <widget class="QLabel" name="autor_lbl">
      <property name="text">
       <string>Autor</string>
      </property>
      <property name="buddy">
       <cstring>author</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="author"/>
    </item>
    <item>
     <widget class="QLabel" name="theme_lbl">
      <property name="text">
       <string>Tema</string>
      </property>
      <property name="buddy">
       <cstring>theme</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="theme"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="confirm">
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
